etl/steps/data/grapher/wb/2024-09-09/food_prices_for_nutrition.meta.yml

```diff
@@ -0,0 +1,105 @@
+tables:
```

Comment:
Why is this not done in garden?

Reply:
When I created the step I thought it would be convenient to separate garden-related metadata from grapher-related metadata. Now we usually do all this in garden. I think it's fine either way.